src/components/common/CollapsibleText/CollapsibleText.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { expect, within } from '@storybook/test';
+
+import { CollapsibleText } from './CollapsibleText';
+
+const meta = {
+  title: 'Common/CollapsibleText',
+  component: CollapsibleText,
+} satisfies Meta<typeof CollapsibleText>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Collapsed = {
+  args: {
+    content:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus scelerisque, nibh eu dapibus dignissim, eros orci tristique ipsum, id interdum ex neque a dui. Quisque pretium aliquam dui, pulvinar venenatis sem consectetur non. Sed felis mi, viverra sit amet blandit sed, vestibulum eu purus. Nulla eget tellus sodales, volutpat nulla in, blandit tellus. Fusce congue vitae elit ac scelerisque. Maecenas fringilla ipsum in enim volutpat dignissim. Aenean convallis urna vel nibh semper faucibus. Nunc non pellentesque nibh. Sed tempor, erat sit amet volutpat placerat, metus quam auctor nunc, in consectetur orci dui quis ligula.',
```

Comment:
You can use the `args` in the meta for default arg values. For `content` and `numberOfLinesToShow` I think it would make sense since you are not changing them in both stories.